ui/Screen/Onboarding/EnterPassphrase.svelte

```diff
@@ -127,10 +127,11 @@
     <h1>Next, you'll need a passphrase.</h1>
 
     <p>
-      This decrypts your key so you can access and publish to the Radicle
-      network. Be sure to store this in a safe place —
+      This protects your Radicle keypair in case your device is compromised.
+      It's used to sign and publish to the Radicle network. Be sure to store
+      this in a safe place —
       <span class="typo-text-bold">
-        there is no way to recover lost keys yet!
+        there is no way to recover your keys without it!
```

Comment:
This insinuates that you can recover your keys? That's not true, right? This copy was saying that if you lose your password, there's nothing you can do and that's still true.

Reply:
"Recover" is meant here as recover from their encrypted form, not recover from loss. We could also say "There is no way to access your keys without it"

Reply:
Yes I understand. But the sentence that was replaced wasn't talking about that, it was talking about what happens when you lose your password. So if you're changing the copy of that sentence, it should still talk about the same topic. Here you changed the copy and the topic, meaning that that original message is lost.

Reply:
Right, because your keys are stored on disk, but your passphrase isn't, so you have to remember it or back it up explicitly, vs. the keys that are already saved locally.

Reply:
But I think it does say the same thing. 

We're saying, make sure to store this passphrase in a safe place, because you can't recover (as in decrypt or access) your keys without it. 

If you lose your keys (delete the file etc...) then you won't be able to recover those either, but you wouldn't be able to recover them with a passphrase, that needs multi-device support. 

So I say we sub 'recover' with 'access' to make this clearer and add another note to the Docs, but besides that, I think it works as is:

```suggestion
        there is no way to access your keys without it!
```

Reply:
:+1: